hermes-api/src/main/java/pl/allegro/tech/hermes/api/jackson/PatchDataSerializer.java

```diff
@@ -0,0 +1,23 @@
+package pl.allegro.tech.hermes.api.jackson;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import pl.allegro.tech.hermes.api.PatchData;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class PatchDataSerializer extends JsonSerializer<PatchData> {
```

Comment:
i didn't want to add it, but then it makes writing integration tests a lot easier - becasue we send PatchData to the server